packages/logic-utils/api.d.ts

```diff
@@ -83,3 +83,23 @@ export function getRootSymbolParent(node: BaseUI5Node): BaseUI5Node | undefined;
  * Return a human-readable string representation of a UI5 type
  */
 export function typeToString(type: UI5Type | undefined): string;
+
+/**
+ * Return the UI5 Class for an XML Element
+ * @param element
+ * @param model
+ */
+export function getClassByElement(
```

Comment:
question: is the verbose `getUI5ClassByXMLElement` name worth it for accuracy?

Reply:
Changed